app/src/main/java/swati4star/createpdf/fragment/ImageToPdfFragment.java

```diff
@@ -791,13 +794,14 @@ private void resetValues() {
                 DEFAULT_PAGE_SIZE));
         mPdfOptions.setPasswordProtected(false);
         mPdfOptions.setWatermarkAdded(false);
+        mSharedPreferences = PreferenceManager.getDefaultSharedPreferences(mActivity);
```

Comment:
why have you added this?

Please do all changes at once. This is taking too much iterations!!!!!

Reply:
then change it?

Reply:
I have moved this lines to resetValues function, it is  right?
And do I need any other changes?

Reply:
There still are changes, but i'll do them & merge PRs.
There have al;ready been so many iterations for a small change

Reply:
OK, apologize for that, and thanks.